vectors.Rmd

```diff
@@ -28,7 +28,7 @@ people.length <- length(people)
 print(people.length)  # [1] 3
 
 numbers <- c(1,2,3,4,5)
-print( length(numbers) )  # [1] 5
+print(length(numbers))  # [1] 5
```

Comment:
The space around the inner expression is provided for clarity (to "group" the operations), which is useful for people who don't have practice reading nested expressions.

Reply:
Perhaps the best way is to restore the spacing but make it clearer that it's for educational purposes and isn't considered good style?